tests/LoggerTest.php

```
@@ -271,10 +259,292 @@ public function testCreateSpanLoggerFromTraceparentFallsBackOnInvalidHeader(): v
+    public function testWithChildSpanEndsTheSpanAndReturnsTheClosureValue(): void
+    {
+        $log = new Logger('svc', 'debug', 'text', "\t", null, null, ['env' => 'prod']);
+
+        $inner = null;
+        $result = $log->withChildSpan('op', ['tenant' => 'acme'], function (Logger $child) use (&$inner) {
+            $inner = $child;
+            return 'returned';
+        });
+
+        $this->assertSame('returned', $result);
+        $this->assertSame(['env' => 'prod', 'tenant' => 'acme'], $this->standingContext($inner));
```

PHPStan (2.2.7)

[warning] Parameter `#1` $log of method Timewave\Logger\Tests\LoggerTest::standingContext() expects Timewave\Logger\Classes\Logger, Timewave\Logger\Classes\Logger|null given.


(argument.type)